### ISSUE TITLE
Create a GSMM infographic

### ISSUE THREAD

**Issue by russelltrow** [Green-Software-Foundation/green-software-maturity-matrix#14]
# Context

Create an [infographic](https://venngage.com/blog/what-is-an-infographic/#what-is-infographic) to provide an easy-to-understand visualisation of the Maturity Matrix

# Acceptance Criteria

- [ ] GSF branded
- [ ] ?

# Actions

- [ ] @aecurrie @Jenya-design have a discussion
- [ ]

**Comment by Jenya-design** [comment]
Hi 
@russelltrow could you please add this task in design project also, so I could see it [on my board](https://github.com/orgs/Green-Software-Foundation/projects/9/views/1)
For some reason I can't do that for this issue

@aecurrie could you please describe the components / steps you would like to show in the infographic
How you're planning to use this image, so you have a specific format in mind?
When you need this piece ready?

I'm travelling starting tomorrow till Dec 29, sorry in advance for delay in reply

Thank you!

**Comment by Jenya-design** [comment]
Hi @aecurrie I hope you had a great holiday season
Could you please give me any additional details for this infographic?

**Comment by aecurrie** [comment]
I think you've already done it! The Maturity matrix matrix (attached).
Having said that I can't find the master copy. You built it as a google
doc? Or slide?

**Comment by Jenya-design** [comment]
@aecurrie sorry, I can't see the attachment
Do you mean this [MM table](https://docs.google.com/presentation/d/1KS3Gzik0iMLPOWLJSGQGOsrihAL0sD48HzksEGBLFCI/edit?usp=sharing)?

**Comment by aecurrie** [comment]
Yes, that's it! BTW I need a print version. Should I just download as SVG?

**Comment by Jenya-design** [comment]
@aecurrie sorry, it's not as straightforward as I though. I think pdf is the best option, though it converts the design into an image. Please find pdf file attached - I converted colors to CMYK for print. It's good for up to 25.4 x 14.29 cm size (A4 paper is 21 x 29.7 cm). Is it ok for you?
All files [here on GSF Public Drive](https://drive.google.com/drive/folders/1a5qEdjI0s4bN1QP2MVvnftGJB4cJbFAB?usp=drive_link)
Let me know if you have any questions or comments

@russelltrow please close this task if it's completed or give me more details if it's a separate infographic
Thank you

[Maturity Matrix table.pdf](https://github.com/Green-Software-Foundation/green-software-maturity-matrix/files/13822511/Maturity.Matrix.table.pdf)

**Comment by aecurrie** [comment]
Thanks!!!
